### ISSUE TITLE
CNN run on permuted MNIST

### ISSUE THREAD

**Issue by tiimoS** [literally-a-placeholder/pattern_recognition#25]
Use the provided training set (permutated MNIST) to train your CNN (from task 2c). Apply the both trained models to classify the test set (permutated MNIST). Compare the results with the results from the normal MNIST. Is there a difference from before? If yes, why? Try to explain what you observe with your own words and the reason of this happening.
Expected Output
• Access to your github so that we can inspect your code.
• Accuracy on the permutated MNIST compared with the results on the normal MNIST. •Analysis of the results in your own words.
